src/vendor/bullet.js

```diff
@@ -55,7 +55,7 @@ export default function bullet () {
       range.enter().append("rect")
           .attr("class", function(d, i) { return "range s" + i; })
           .attr("width", w0)
-          .attr("height", height)
+          .attr("height", Math.abs(height))
```

Comment:
I don't think we should edit the extension itself, we should prevent it from ever happening/sending it to here

Reply:
It is Ok to edit the bullet.js.
But i agree this should be handled before sending the value to it.

Reply:
I understand, you did an update on bullet on a different branch, right? I meant I'd prefer we fixed it on our end before we edit bullet.js but that's an opinion not a rule 😄

Reply:
Yes i have.
As i said i do agree with your point, if it could be handled before being sent to bullet.js then it should be done :)

Reply:
The previous comment is about that (L132) . If we tackle that problem there, we can skip this edit